CONTRIBUTING.md

```diff
@@ -0,0 +1,130 @@
+<!-- markdownlint-disable MD033 MD013 -->
+# Contributing to Casper - Guideline Document
+
+Welcome! If you are visiting this page, you're likely interested in contributing to the development of Casper. We are thrilled to have you on board.
+
+Casper is a strong advocate of community participation and warmly welcomes contributions in various forms.
+
+There is always a need for improvement in our documentation, creating tutorials, and writing tests. You can contribute to Casper by engaging in development, code reviews, and enhancing documentation.
+
+## Code of Conduct
+
+The Casper [Code of Conduct](https://github.com/casper-ecosystem/.github/blob/main/profile/CODE_OF_CONDUCT.md) governs the conduct expected in this project. By participating, you agree to adhere to this code.
+
+## How Can I Contribute?
+
+### 1. Reporting Bugs
+
+If you encounter issues with `casper-node`, please file an issue in the repository using the specified bug report template. Providing detailed information will help expedite the resolution.
+
+#### Before Submitting A Bug Report
+
+- Confirm the issue is related to `casper-node` by consulting the documentation and FAQs.
+- Ensure that the problem has not been previously reported by searching through existing issues.
+- If a similar issue exists but remains unresolved, consider contributing to that thread instead of starting a new one.
+
+#### How Do I Submit A (Good) Bug Report?
+
+A well-crafted bug report should clearly outline the problem by following these steps:
+
+- **Use a descriptive title** to help pinpoint the issue.
+- **Detail the exact steps to reproduce the problem** so that we can replicate it.
+- **Provide specific examples**, such as code snippets, links to files, or GitHub projects that are part of the test case.
+- **Describe the observed behavior** and how it differed from your expectations.
+
+### 2. Code Contributions
+
+Whether fixing bugs or developing new features, your contributions are highly valued.
+
+If you are new and looking for simpler tasks, explore the issues labeled as `good first issue`.
+
+You can also check out `beginner` and `help-wanted` issues.
+
+Here’s how you can contribute code:
+
+#### Steps to Submit Code
```

Comment:
Why is the formatting so different than what was submitted here: https://github.com/casper-network/.github/pull/1#issuecomment-2073024912. Could you please make them consistent?

Reply:
This comment still needs to be addressed.